app/controllers/samples_controller.rb

```diff
@@ -172,7 +172,7 @@ def show
 
     background_id = check_background_id(@sample)
     @report_page_params = { pipeline_version: @pipeline_version, background_id: background_id } if background_id
-    if @pipeline_run && (((@pipeline_run.remaining_reads.to_i > 0 || @pipeline_run.finalized?) && !@pipeline_run.failed?) || @pipeline_run.report_ready?)
+    if @pipeline_run && (((@pipeline_run.remaining_reads.to_i > 0 || @pipeline_run.results_finalized?) && !@pipeline_run.failed?) || @pipeline_run.report_ready?)
```

Comment:
Will pipeline_run.failed? be true when all batch jobs completed successfully but some results failed to load?

Reply:
ok I changed it so that the answer is yes